rubble/src/att/mod.rs

```diff
@@ -622,6 +991,8 @@ impl<A: AttributeProvider> AttributeServer<A> {
                 code: ErrorCode::InvalidPdu,
                 handle: AttHandle::NULL,
             }),
+
+            _ => unimplemented!("Tried to process unknown request: {:?}", pdu.params),
```

Comment:
We should write down the required ATT opcodes. For those that aren't required we can just log this and return a "not implemented" response.